ister.py

```diff
@@ -66,6 +66,90 @@
 
 LOG = None
 
+def extract_full_lines(text):
+    """Extract full lines from string 'text'. Return a tuple containing 2 elements
+    - the full lines and the last partial line.
+    """
+
+    full, partial = ("", "")
+    for line_match in re.finditer("(.*\n)|(.+$)", text):
+        if line_match.group(2):
+            partial = line_match.group(2)
+            break
+        full += line_match.group(1)
+    return (full, partial)
+
+def stream_fetcher(info, streamid):
```

Comment:
This and extract_full_lines were unused and needed imports that weren't added at the time. I think I reviewed it correctly but I'm not sure if this was intended to be included here or not.

Reply:
I'll check and if this is the case I'll amend the patches and will make sure that every patch adds the import it requires. I was doing a lot of rebase -i and could make a mistake when splitting patches.